app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -28,6 +28,8 @@
     <p><a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Find out more about training to teach if you do not have a degree</a>.</p>
     <h3>If you do not want to get a degree</h3>
     <p>You do not need a degree and QTS to <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teach in further education</a> or to <a href="/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers">teach in early years</a>. So if you want to become a teacher, but are not able to get a degree, you can explore these options.</p>
+    <h3>If you're a teaching assistant</h3>
+    <p>If you're a teaching assistant, your valuable classroom experience will make you a desirable candidate for teacher training. You'll still need a degree and QTS to teach in most schools, which you can achieve through the same routes as other career changers.</p>    <p>If you've been a teaching assistant for 2 years and have taught classes independently, you may be eligible for an <a href="/train-to-be-a-teacher/assessment-only-route-to-qts">assessment only route to QTS</a>.</p>
```

Comment:
The second para needs to start on a new line in the code (it's OK on the front end though)